apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/index.tsx

```diff
@@ -309,7 +311,39 @@ export default function DocumentEditor({
                   providers={providers}
                   disabledMetadataSelectors={isMerged}
                   title='Prompt editor'
-                  rightActions={<UpdateToPromptLButton document={document} />}
+                  rightActions={
+                    <>
+                      <Tooltip
+                        trigger={
+                          <Link
+                            href={
+                              ROUTES.projects
+                                .detail({ id: project.id })
+                                .commits.detail({ uuid: commit.uuid })
+                                .history.detail({
+                                  uuid: document.documentUuid,
+                                }).root
+                            }
+                          >
+                            <Button
+                              variant='outline'
+                              size='small'
+                              className='h-7'
```

Comment:
Why hardcoded height?

Reply:
![image](https://github.com/user-attachments/assets/d7919a11-8b61-4aa3-be46-31af84fed536)

Reply:
Why h-7? Is not part of our design system no?

Reply:
For some reason the `DropdownMenuTrigger` has `5px` of vertical padding (?)

Reply:
I don't like this tbh

Reply:
I can't exactly point where is this padding being added